MapboxNavigation/Style.swift

```diff
@@ -227,6 +227,11 @@ open class Style: NSObject {
     public var trafficSevereColor: UIColor?
     
     /**
+     Describes the situations in which the style should be used. By default, the style will be used during the daytime.
+     */
+    public var styleType: StyleType = .daytimeStyle
```

Comment:
@frederoni how should a map style be linked to a UI style? Do you think we should add a new prop to Style that is the associated map style url?